sdk/ai/Azure.AI.Projects/README.md

```diff
@@ -210,6 +212,110 @@ Agent agent = agentResponse.Value;
 With a file ID association and a supported tool enabled, the agent will then be able to consume the associated
 data when running threads.
 
+#### Create Agent with Enterprise File Search
+
+We can upload file to Azure as it is shown in the example, or use the existing Azure blob storage. In the code below we demonstrate how this can be achieved. First we upload file to azure and create `VectorStoreDataSource`, which then is used to create vector store. This vector store is then given to the `FileSearchTool` constructor.
+
+```C# Snippet:CreateVectorStoreBlob
+var ds = new VectorStoreDataSource(
+    assetIdentifier: blobURI,
+    assetType: VectorStoreDataSourceAssetType.UriAsset
+);
+var vectorStoreTask = await client.CreateVectorStoreAsync(
+    fileIds: null,
+    name: "sample_vector_store",
```

Comment:
Is the above not the default? do you need to show `fileIds: null,`?

Reply:
Fixed